src/pages/registration/FilesAndLicensePanel.tsx

```diff
@@ -416,6 +427,31 @@ export const FilesAndLicensePanel = ({ uppy }: FilesAndLicensePanelProps) => {
           </Box>
         ))}
       </Modal>
+
+      <Modal
+        headingText={t('common.version')}
+        open={isVersionModalOpen}
+        onClose={toggleVersionModal}
+        maxWidth="sm"
+        dataTestId={dataTestId.registrationWizard.files.versionModal}>
+        <Box sx={{ display: 'flex', flexDirection: 'column', gap: '1rem' }}>
+          <Typography>{t('registration.files_and_license.version_helper_text')}</Typography>
+          <Typography>
+            <b>{t('registration.files_and_license.accepted_version')}</b>
+            {': '}
+            {t('registration.files_and_license.version_accepted_helper_text')}
+          </Typography>
```

Comment:
Dette er kanskje enklere? Litt teit den med å legge til `{': '}` da, men greit nok. Kunne ev løst det i i18n også, som kanskje hadde vært aller best (men litt krøkk det også)
```suggestion
          <Typography paragraph>
            <Box component="span" sx={{ fontWeight: 700 }}>
              {t('registration.files_and_license.accepted_version')}
              {': '}
            </Box>
            {t('registration.files_and_license.version_accepted_helper_text')}
          </Typography>
```

Reply:
Fortsatt litt teit den med å legge til `{': '}` da men. Kunne ev løst det i i18n også, som kanskje hadde vært aller best, men litt krøkk det også, når første element skal være bold. Men det er fullt løsbart da, om du vil undersøke

Reply:
Prøvde en løsning i i18n, men er ikke sikker på hvor mye bedre den er.

Reply:
Syns det ble veldig bra! Det som er den anbefalte måten, for da kan oversetter fortsatt kontrollere hele teksten i en nøkkel